ceph_iscsi_config/lun.py

```diff
@@ -574,6 +581,38 @@ def activate(self):
             if client_err:
                 raise CephiSCSIError(client_err)
 
+    def _erasure_pool_check(self):
+        if not self.ecpool:
+            return None
+
+        data, err = run_shell_cmd(
+            "ceph -n {client_name} --conf {cephconf} osd metadata --format=json".
+            format(client_name=settings.config.cluster_client_name,
+                   cephconf=settings.config.cephconf))
+        if err:
+            self.logger.error("Cannot get the objectstore type")
+            return err
+        bluestore = False
+        for _osd in json.loads(data):
+            store_type = _osd['osd_objectstore']
+            self.logger.debug(f"pool ({self.pool}) objectstore type is ({store_type})")
+            if store_type == 'bluestore':
+                bluestore = True
+                break
+
+        if not bluestore:
+            return None
+
+        data, err = run_shell_cmd(
+            "ceph -n {client_name} --conf {cephconf} osd pool get {pool} allow_ec_overwrites".
+            format(client_name=settings.config.cluster_client_name,
+                   cephconf=settings.config.cephconf, pool=self.ecpool))
+        if err:
+            self.logger.error(f"Cannot get allow_ec_overwrites from pool ({self.pool})")
+            return err
+        self.logger.debug(f"erasure pool ({self.pool}) allow_ec_overwrites is enabled")
```

Comment:
The actual check seems to be missing.  What if `allow_ec_overwrites` is disabled?

Reply:
Sorry, I didn't get this, could you explain more ?

Reply:
If the `allow_ec_overwrites` is disabled, it should abort the image creation should fail.

Reply:
Okay, get it. Will fix it.